.changeset/wild-maps-brake.md

```diff
@@ -0,0 +1,7 @@
+---
+'@powersync/service-rsocket-router': patch
+'@powersync/service-errors': patch
+'@powersync/service-core': patch
```

Comment:
I think this should be a `minor` bump for `@powersync/service-core`?

FYI the `@powersync/service-core` and `@powersync/service-image` "package" (used to publish the DockerHub Docker image) are configured for fixed (linked) versioning. I think it would be nice to add an explicit changeset for `@powersync/service-image` mentioning that the new release adds support for JSON websocket request payloads (or something along those lines)

Reply:
I've made it a `minor` bump for `service-core` and I've also added a changelog entry for the service image.